CHANGELOG.md

```diff
@@ -77,7 +77,7 @@
   - support for binary arithmetic and comparisons between `Timedelta` values and numeric values.
   - support for lazy `TimedeltaIndex`.
   - support for `pd.to_timedelta`.
-  - support for `GroupBy` aggregations `min`, `max`, `mean`, `idxmax`, `idxmin`, `std`, `sum`, `median`, `count`, `any`, `all`, `size`, `nunique`.
+  - support for `GroupBy` aggregations `min`, `max`, `mean`, `idxmax`, `idxmin`, `std`, `sum`, `median`, `count`, `any`, `all`, `size`, `nunique`, `head`, `tail`, `first`, `last`, `aggregate`.
```

Comment:
nit: `first` and `last` are filtrations, not aggregations, so we should list them separately

Reply:
I don't think this is necessary as they all fit into the paradigm `groupby.<func>()` but can separate if you think it's better.

Reply:
I'd prefer to be precise and separate them because pandas explicitly says `head` and `tail` are not aggregations: https://pandas.pydata.org/docs/user_guide/groupby.html#filtration

Reply:
SG, will do.